<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1980</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LabelingTool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="b_label0">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>70</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Label 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_label1">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>130</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Label 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_label2">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>190</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Label 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_addlabel">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>780</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Label</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1980</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuMen">
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Men√π</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew_Project"/>
    <addaction name="actionLoad_Project"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuMen"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project...</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionLoad_Project">
   <property name="text">
    <string>Load Project...</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
